core/src/com/po/fuck/Constants.java

```diff
@@ -21,9 +21,10 @@ public final class Constants {
     public static final int ENEMY_TEAM_TAG = 0;
     public static final int PLAYER_TEAM_TAG = 1;
 
-    public static final float spriteCoinPositionX = 80;
-    public static final float spriteCoinPositionY = 45f;
-    public static final float coinNumberPosX = 35;
-    public static final float widthOfCoinNumber = 17;
-    public static final float coinNumberPosY = GAME_HEIGHT - 30;
+    public static final float SPRITE_COIN_POSITION_X = 80;
```

Comment:
Probably can use `Vector2` for the position, also words order should be `COIN_SPRITE_POSITION`.